### PR TITLE
added jpgpi250/DoH entry

### PR DESCRIPTION
**Pull request by jpgpi250**
Signed-off-by: jpgpi250 <jpgpi250@gmail.com>

as suggested by "Jason lsh", [here](https://forum.suricata.io/t/sid-allocation-for-ruleset-i-wish-to-share/2939/4), adding the required lines to create a new entry.

### CODE REVIEW

**Comment by jpgpi250** [comment]
@jasonish
Anything wrong with my PR? rejected?

File: index.yaml [hunk 1]
```diff
@@ -143,6 +143,18 @@ sources:
     support-url: https://discord.com/channels/911231224448712714/911238451842666546
     url: https://ti.stamus-networks.io/open/stamus-lateral-rules.tar.gz
 
+  jpgpi250/DoH:
```
Comment (jasonish) [inline index.yaml]: I'll probably make this all lower case for consistency with the rest of the index.

**Comment by jasonish** [comment]
> @jasonish Anything wrong with my PR? rejected?

Looks good. Just a little behind.

**Comment by jasonish** [comment]
Actually, I just realized these are all `reject` rules.  Will need to think about how this makes sense in the more general case.

File: index.yaml (hunk 1 above)
Comment (jpgpi250) [reply index.yaml]: changes as requested

**Pull request closed without merging**